### ISSUE TITLE
macOS 10.14 (Mojave) Cmd + T does not work

### ISSUE THREAD

**Issue by mik9** [keeweb/keeweb#953]
Thanks for taking the time to contribute!

Please, read https://github.com/keeweb/keeweb/wiki/FAQ before submitting a new issue and fill in the following (if applicable):

- Open KeeWeb -> Choose credentials -> Cmd + T
- Nothing happened
- 1.6.3
- your user-agent (from Settings/Help section)
KeeWeb v1.6.3 (cded8a4, 2017-12-11)
Environment: electron v1.7.9
User-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_0) AppleWebKit/537.36 (KHTML, like Gecko) KeeWeb/1.6.3 Chrome/58.0.3029.110 Electron/1.7.9 Safari/537.36

- please, open dev tools in your browser and attach output log from Console tab 
    (if you are using desktop app, devtools can be opened from Settings/General/Advanced)

```
2018-06-30T11:44:28.848Z [auto-type] Auto type event n {...}
index.html:1 2018-06-30T11:44:28.850Z [auto-type] Hide window
index.html:1 2018-06-30T11:44:28.957Z [auto-type] Start {USERNAME}{TAB}{PASSWORD}{ENTER}
index.html:1 2018-06-30T11:44:28.958Z [auto-type] Parsed [op:USERNAME,op:TAB,op:PASSWORD,op:ENTER]
index.html:1 2018-06-30T11:44:28.959Z [auto-type] Resolved [****,key:tab,[*,*,*,*,*,*,*,*,*,*,*,*,*,*,*,*,*,*,*,*,*,*,*,*],key:enter]
index.html:1 2018-06-30T11:44:29.172Z [launcher] spawn /Applications/KeeWeb.app/Contents/Resources/helper/darwin/KeeWebHelper: 0, 213ms
index.html:1 2018-06-30T11:44:29.173Z [auto-type] Complete 215ms
```

**Comment by antelle** [comment]
Hi! We don't support this version yet, but a fix would be very welcome.

**Comment by mik9** [comment]
Not a macOS developer, but this might be related to:
https://forums.developer.apple.com/thread/103992

**Comment by mik9** [comment]
Logs from Console:

<img width="854" alt="22 44 19" src="https://user-images.githubusercontent.com/244760/42138082-603fb6c6-7d80-11e8-805b-e17ccaf35d4d.png">

**Comment by mik9** [comment]
After some more research I went to `System Settings -> Security & Privacy -> Privacy -> Accessibility` and found KeeWeb there in unchecked state. After I removed it from list on next `Cmd + T` macOS asked me to add KeeWeb to the Accessibility list and `Cmd + T` start working after this.

I'm pretty sure I didn't changed anything after update to 10.14.

**Comment by antelle** [comment]
Interesting, thanks for the investigation!

**Comment by mik9** [comment]
A bit more info: after uncheck/check Keeweb in `System Settings -> Security & Privacy -> Privacy -> Accessibility` `Cmd + T` works only one time. Then I need to uncheck/check again for next time.

**Comment by mstarke** [comment]
macOS 10.14 Mojave prevents applications from sending events to other applications without the user's consent (or altogether). This is sort of breaking a lot of tools and hopefully will be fixed in the final version of Mojave. Currently there's an entitlement that Apple uses to allow this. This behaviour is enforced to all applications (sandboxed and non-sandboxed as well)

**Comment by ddotlic** [comment]
@antelle @mik9 Final Mojave version tested and after enabling the KeeWeb in the `System Settings -> Security & Privacy -> Privacy -> Accessibility` things work fine for me.

**Comment by mik9** [comment]
Yes, I can also confirm. On final Mojave after I manually enabled KeeWeb in `Accessibility` it works fine.
